[2024-06-28T01:17:57.408+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T01:17:57.453+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: download_data.download_stock_data scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-06-28T01:17:57.464+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: download_data.download_stock_data scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-06-28T01:17:57.465+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-28T01:17:57.483+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_stock_data> on 2024-06-27 00:00:00+00:00
[2024-06-28T01:17:57.495+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=55) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-28T01:17:57.497+0000] {standard_task_runner.py:63} INFO - Started process 67 to run task
[2024-06-28T01:17:57.498+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'download_data', 'download_stock_data', 'scheduled__2024-06-27T00:00:00+00:00', '--job-id', '938', '--raw', '--subdir', 'DAGS_FOLDER/download_data.py', '--cfg-path', '/tmp/tmpggpg_hcl']
[2024-06-28T01:17:57.502+0000] {standard_task_runner.py:91} INFO - Job 938: Subtask download_stock_data
[2024-06-28T01:17:57.518+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-06-28T01:17:57.598+0000] {task_command.py:426} INFO - Running <TaskInstance: download_data.download_stock_data scheduled__2024-06-27T00:00:00+00:00 [running]> on host 5daea60ed64a
[2024-06-28T01:17:57.611+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:356: DeprecationWarning: The sql_engine_encoding option in [core] has been moved to the sql_engine_encoding option in [database] - the old setting has been used, but please update your config.
  engine_args["encoding"] = conf.get("database", "SQL_ENGINE_ENCODING", fallback="utf-8")

[2024-06-28T01:17:57.804+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='download_data' AIRFLOW_CTX_TASK_ID='download_stock_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T00:00:00+00:00'
[2024-06-28T01:17:57.806+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T01:17:57.831+0000] {download_data.py:18} INFO - Starting data download for AAPL for period 1y
[2024-06-28T01:17:57.832+0000] {logging_mixin.py:188} INFO - yfinance: pandas_datareader support is deprecated & semi-broken so will be removed in a future verison. Just use yfinance.
[2024-06-28T01:17:58.755+0000] {logging_mixin.py:188} WARNING - [*********************100%%**********************]  1 of 1 completed
[2024-06-28T01:17:58.759+0000] {download_data.py:21} INFO - Data download complete, number of rows downloaded: 252
[2024-06-28T01:17:58.769+0000] {download_data.py:23} INFO - Data saved to /opt/***/data/data.csv
[2024-06-28T01:17:58.770+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-28T01:17:58.772+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T01:17:58.795+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=download_data, task_id=download_stock_data, run_id=scheduled__2024-06-27T00:00:00+00:00, execution_date=20240627T000000, start_date=20240628T011757, end_date=20240628T011758
[2024-06-28T01:17:58.887+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T01:17:58.927+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-28T01:17:58.929+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-28T10:14:51.549+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T10:14:51.573+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: download_data.download_stock_data scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-06-28T10:14:51.582+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: download_data.download_stock_data scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-06-28T10:14:51.583+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-28T10:14:51.598+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_stock_data> on 2024-06-27 00:00:00+00:00
[2024-06-28T10:14:51.622+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=45) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-28T10:14:51.625+0000] {standard_task_runner.py:63} INFO - Started process 48 to run task
[2024-06-28T10:14:51.634+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'download_data', 'download_stock_data', 'scheduled__2024-06-27T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/download_data.py', '--cfg-path', '/tmp/tmpv54e0zl7']
[2024-06-28T10:14:51.647+0000] {standard_task_runner.py:91} INFO - Job 35: Subtask download_stock_data
[2024-06-28T10:14:51.667+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-06-28T10:14:51.704+0000] {task_command.py:426} INFO - Running <TaskInstance: download_data.download_stock_data scheduled__2024-06-27T00:00:00+00:00 [running]> on host 0d0315ca3a10
[2024-06-28T10:14:51.707+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:356: DeprecationWarning: The sql_engine_encoding option in [core] has been moved to the sql_engine_encoding option in [database] - the old setting has been used, but please update your config.
  engine_args["encoding"] = conf.get("database", "SQL_ENGINE_ENCODING", fallback="utf-8")

[2024-06-28T10:14:51.779+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='download_data' AIRFLOW_CTX_TASK_ID='download_stock_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T00:00:00+00:00'
[2024-06-28T10:14:51.780+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T10:14:51.815+0000] {download_data.py:18} INFO - Starting data download for AAPL for period 1y
[2024-06-28T10:14:51.815+0000] {logging_mixin.py:188} INFO - yfinance: pandas_datareader support is deprecated & semi-broken so will be removed in a future verison. Just use yfinance.
[2024-06-28T10:15:03.628+0000] {base.py:126} ERROR - Failed to get ticker 'AAPL' reason: HTTPSConnectionPool(host='query2.finance.yahoo.com', port=443): Max retries exceeded with url: /v8/finance/chart/AAPL?range=1d&interval=1d&crumb=j2bED3WNXcA (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0xffff6658f290>: Failed to resolve 'query2.finance.yahoo.com' ([Errno -3] Temporary failure in name resolution)"))
[2024-06-28T10:15:06.093+0000] {logging_mixin.py:188} WARNING - [*********************100%%**********************]  1 of 1 completed
[2024-06-28T10:15:06.106+0000] {download_data.py:21} INFO - Data download complete, number of rows downloaded: 252
[2024-06-28T10:15:06.156+0000] {download_data.py:23} INFO - Data saved to /opt/***/data/data.csv
[2024-06-28T10:15:06.160+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-28T10:15:06.165+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T10:15:06.301+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=download_data, task_id=download_stock_data, run_id=scheduled__2024-06-27T00:00:00+00:00, execution_date=20240627T000000, start_date=20240628T101451, end_date=20240628T101506
[2024-06-28T10:15:06.392+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T10:15:06.477+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-28T10:15:06.485+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
